README.md

```diff
@@ -2,10 +2,29 @@
 
 1. Install Rust (https://www.rust-lang.org/tools/install)
 
+#### Versions
+this was tested on:
+
+| dependency    | version
+| ------------- |:-------------:|
+| Rust | 1.45.0 |
+| Cargo | 1.45.0 |
+| Actix-Web | 0.7.19 |
+| Sentry SDK | 0.19.0 |
+
+
 ## Run
 
 1. go into the directory that has the Cargo.toml file in it
 2. run "cargo run"
 3. go to URL http://localhost:3001/checkout
```

Comment:
@sentryadam0000345 we don't actually visit URL /checkout in the browser or Postman
"go to..."makes it sound like they might be expecting a web page to load (like when we say 'go to sentry.io')
how about 'hit'
3. hit URL http://localhost:3001/handled
or
3. http://localhost:3001/handled
4. http://localhost:3001/unhandled
5. http://localhost:3001/checkout w/ payload from sentry-demos/react

Reply:
^ indicates that /handled /unhandled are things they can hit however they;d like (browser, postman) and /checkout is clearly something they won't get via a simple browser GET or postman GET, w/out knowing its a POST and having the right body